go-selinux/selinux_linux.go

```diff
@@ -81,6 +81,11 @@ var (
 	// for getPolicyRoot
 	getPolicyRootOnce sync.Once
 	policyRoot        string
+
+	// for loadLabels
+	labels                  map[string]string
+	privContainerMountLabel string
+	loadLabelsOnce          sync.Once
```

Comment:
1. done
2. I'd like it to stay as is, because otherwise it would be less clear that `privContainerMountLabel` is only initialized together with `labels` (I _think_ that we can't move that to `PrivContainerMountLabel()` because of the reservation).

Reply:
Re: 2. OTOH the code is not very complex and it is easy to figure out how `privContainerMountLabel` is initialized.

Done.